docs/spec/lcd/api.md

```diff
@@ -0,0 +1,847 @@
+# Cosmos Hub (Gaia) LCD API
```

Comment:
Per @nylira This doc is missing the following endpoints:

```
/net_info
/dump_consensus_state
/consensus_state
/subscribe
/unsubscribe
```

Also a couple of notes in his issue about address compatibility